core/trie/src/lib.rs

```diff
@@ -152,6 +162,21 @@ pub fn child_trie_root<H: Hasher, I, A, B>(_storage_key: &[u8], input: I) -> Vec
 	trie_root::<H, _, _, _>(input).as_ref().iter().cloned().collect()
 }
 
+pub fn child_trie_root2<H: Hasher, I, A, B>(_storage_key: &[u8], input: I) -> H::Out where
```

Comment:
Are this actually used? The reason for `child_trie_root` to return `Vec<u8>` is so that it can support multi-hash -- some hash algorithms applied to trie hash in the future might use a different length.

`H` is only the default hasher here, meaning a multi-hash if branch can ignore that generic parameter.

Reply:
Not used anymore, I will remove it.